static/js/signal.js

```diff
@@ -0,0 +1,32 @@
+window.onload = function() {
+  updateSignalLoop()
```

Comment:
Would it be simpler to use `setTimeout` or `setInterval`?

- https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout
- https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval

Reply:
Synce function is async I think can just call updateSignal()